docs/api/tutorials/container.md

```diff
@@ -0,0 +1,31 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Container
+
+## Why Would You Use Containers?
+
+The Container entity represents a logical grouping of entities, such as datasets, data processing instances, or even other containers. It helps users organize and manage metadata in a hierarchical structure, making it easier to navigate and understand relationships between different entities.
+
+### Goal Of This Guide
```

Comment:
we probably need to explain how containers relate to other entity types

Reply:
added explanations